src/entities/Basket/Basket.interface.ts

```diff
@@ -0,0 +1,14 @@
+import * as mongoose from 'mongoose';
+
+export interface Basket{
+    _id: mongoose.Types.ObjectId,
+    createdByUserId: mongoose.Types.ObjectId,
```

Comment:
CreatedByUserID interface will be used while creating validation funtion - and we assign user automatically, using JSON web token. Thus, it should be optional or even deleted. 
Moreover, I think it would be better non to use createdByUserId ect. - just createdByUser (we will call populate, so eventually we will receive not only ID, but the whole user) :)